modules/@apostrophecms/i18n/ui/apos/components/AposI18nLocalize.vue

```diff
@@ -851,7 +854,7 @@ export default {
                   locale: locale.name
                 }
               });
-              if (result.redirectTo) {
+              if (this.redirect && result.redirectTo) {
```

Comment:
if you don't want to redirect, why do we return a `redirectTo` property? We then add a new vue prop to skip the redirect if it's there. It works ok, I'm confused.